sklearn/metrics/_scorer.py

```diff
@@ -714,6 +714,9 @@ def make_scorer(
     >>> from sklearn.svm import LinearSVC
     >>> grid = GridSearchCV(LinearSVC(), param_grid={'C': [1, 10]},
     ...                     scoring=ftwo_scorer)
+
+    For an example on how to use multiple scorers at the same time see
+    :ref:`sphx_glr_auto_examples_model_selection_plot_multi_metric_evaluation.py`.
```

Comment:
So multi metric scoring is interesting, as in, the class itself is not public, and `make_scorer` could potentially support it, but it doesn't (ref https://github.com/scikit-learn/scikit-learn/issues/31540)

The example doesn't really show how to do multi metric scoring with `make_scorer`, so the location of this link doesn't seem right to me.

We do have a link to the example in the user guide, but I think there can also be a link in the API docs of gridsearch classes to show how users can do multi metric scoring, instead of here.